src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/metrics/javancss/InputJavaNCSSRecordsAndCompactCtors.java

```diff
@@ -0,0 +1,180 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.metrics.javancss; // violation for file
+
+import java.time.LocalDateTime;
+
+/* Config:
+ * methodMaximum = 7
+ * classMaximum = 3
+ * fileMaximum = 2
+ * recordMaximum = 4
+ */
+// -@cs[AbbreviationAsWordInName] We can not change it as,
```

Comment:
are you sure we need this ? we suppose to not run any other Checks over inputs.

Reply:
You are correct, fixed.